samza-core/src/main/java/org/apache/samza/execution/ExecutionPlanner.java

```diff
@@ -57,73 +57,75 @@ public ExecutionPlanner(Config config, StreamManager streamManager) {
     this.streamManager = streamManager;
   }
 
-  public ProcessorGraph plan(StreamGraph streamGraph) throws Exception {
-    // create physical processors based on stream graph
-    ProcessorGraph processorGraph = createProcessorGraph(streamGraph);
+  public JobGraph plan(StreamGraph streamGraph) throws Exception {
```

Comment:
Let me be lazy and keep JobGraph for now :). If we decide to go with ExecutionPlan, I will do another renaming change.